xline/src/server/command.rs

```diff
@@ -263,16 +294,19 @@ where
         cmd: &Command,
         index: LogIndex,
         need_run: bool,
-    ) -> Result<SyncResponse, ExecuteError> {
+        revision: Option<i64>,
+    ) -> Result<SyncResponse, Self::Error> {
```

Comment:
`SyncResponse` can be determined before calling `after_sync`. We don’t need to return it from `after_sync`, BTW, `SyncResponse` and `PrepareResponse` both `revision` of a same command, Do we need them all?

Reply:
We don't have a response named`PrepareResponse`. The reversion generated by the `prepare` method is transparent to the user. The user still relies on the `SyncResponse` to get the revision.